### PR TITLE
chore: Minimal set of permissions to deploy che on openshift

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
chore: Minimal set of permissions to deploy che on openshift

## What issues does this pull request fix or reference?
https://github.com/eclipse-che/che/issues/23056

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/permissions-to-install-che-on-openshift-using-the-web-console.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+:_content-type: REFERENCE
+:description: Minimum set of permissions required to install {prod-short} on OpenShift using the web console
+:keywords: permissions, openshift, installing
+:navtitle: Permissions to install {prod-short} on OpenShift using web console
+
+[id="permissions-to-install-che-on-openshift-using-the-web-console"]
+= Permissions to install {prod-short} on OpenShift using the web console
+
+Here you can find the minimal set of permissions required to install {prod-short} on an OpenShift cluster using the web console:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/permissions-to-install-che-on-openshift-using-the-web-console.adoc]:
```suggestion
Below is the minimal set of permissions required to install {prod-short} on an OpenShift cluster using the web console:
```

File: modules/administration-guide/pages/permissions-to-install-che.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+:description: Permissions to install {prod}
+:keywords: administration-guide, installing, permissions
+:navtitle: Permissions to install Che
+:page-aliases:
+
+[id="permissions-to-install-che"]
+= Permissions to install {prod-short}
+
+This section describes the permissions required to install {prod} on different {kubernetes} clusters.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/permissions-to-install-che.adoc]:
```suggestion
Learn about the permissions required to install {prod} on different {kubernetes} clusters.
```

File: modules/administration-guide/pages/permissions-to-install-che-on-openshift-using-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
+:_content-type: REFERENCE
+:description: Minimum set of permissions required to install {prod-short} on OpenShift using CLI
+:keywords: permissions, openshift, installing, cli
+:navtitle: Permissions to install {prod-short} on OpenShift using CLI
+
+[id="permissions-to-install-che-on-openshift-using-cli"]
+= Permissions to install {prod-short} on OpenShift using CLI
+
+Here you can find the minimal set of permissions required to install {prod-short} on an OpenShift cluster using {prod-cli}:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/permissions-to-install-che-on-openshift-using-cli.adoc]:
```suggestion
Here you can find the minimal set of permissions required to install {prod-short} on an OpenShift cluster using {prod-cli}:
```
```suggestion
Below is the minimal set of permissions required to install {prod-short} on an OpenShift cluster using {prod-cli}:
```

**Review by deerskindoll — COMMENTED** [review]
left a couple of small suggestions

**Review by deerskindoll — APPROVED** [review]
lgtm

**Review by ibuziuk — APPROVED** [review]

**Pull request merged**